Packs/Malwarebytes/Integrations/Malwarebytes/Malwarebytes.py

```diff
@@ -0,0 +1,1516 @@
+            outputs={
+                'Malwarebytes.SA(val.Machine_ID == obj.Machine_ID)': {
+                    'Machine_ID': machine_id
+                }
+            },
+            raw_response=[]
+        )
+    else:
+        return_error(message="HTTP Error: " + str(response.status_code) + " on initiating Close SA Incident action",
+                     error='')
+
+
+def get_sa_activities_command(account_id, client_id, auth_token, hostname, path):
+    suspicious_activities = get_suspicious_activities(account_id, client_id, auth_token)
+    if hostname:
+        Filtered_SA = []
+        for i in suspicious_activities:
+            if i['pc_hostname'] == hostname:
+                Filtered_SA.append(i)
+        if len(Filtered_SA) != 0:
+            return_outputs(
+                readable_output=tableToMarkdown(f'Suspicious Activites found for the host: {hostname}', Filtered_SA),
+                outputs={
+                    'Malwarebytes.Endpoint(val.Suspicious_Activities == obj.Suspicious_Activities)': {
+                        'Suspicious_Activities': Filtered_SA
+                    }
+                },
+                raw_response=Filtered_SA
+            )
+        else:
+            return_error(message="No Suspicious Activites found for the host: " + str(hostname), error='')
+    elif path:
+        Filtered_SA = []
+        for i in suspicious_activities:
+            if i['path'] == path:
+                Filtered_SA.append(i)
+        if len(Filtered_SA) != 0:
+            return_outputs(
+                readable_output=tableToMarkdown(f'File Path: {path} is found on the hosts', Filtered_SA),
+                outputs={
+                    'Malwarebytes.Endpoint(val.Suspicious_Activities == obj.Suspicious_Activities)': {
+                        'Suspicious_Activities': Filtered_SA
+                    }
+                },
+                raw_response=Filtered_SA
+            )
+        else:
+            return_error(message="File Path: " + str(path) + " is not found on any hosts.", error='')
+    else:
+        return_error(message='Please enter hostname or path', error='')
+
+
+def main():
+    '''CONSTANTS'''
+    account_id = demisto.params().get('accountid')
+    client_id = demisto.params().get('clientid')
+    client_secret = demisto.params().get('clientsecret')
+
+    auth_token = get_token(client_id, client_secret, account_id)
+
+    try:
+        # The command demisto.command() holds the command sent from the user.
+        LOG(f'command is {demisto.command()}')
+        if demisto.command() == 'test-module':
+            # This is the call made when pressing the integration test button.
+            status = test_connectivity(account_id, client_id, auth_token)
+            if status is True:
+                demisto.results('ok')
+                sys.exit(0)
+            else:
+                demisto.results('Connection Failed! Please check your Credentials')
```

Comment:
it is better if we show the actual api response here, so it will be more indicative